Create application_vnd.openminds.schema+json.jsonld

lzehl opened the pull request:
We may also want to create one for the schema templates (TBD). 

Not sure how to handle versioning here. The syntax should be separately versioned (if we would define it as a schema format). From now to the start we had at least one version increase, from the old to the new pipeline (where we separated fully build schemas from schema templates more clearly through different file extensions).

@apdavison @olinux what do you think? (one review is sufficient I think)

apdavison commented:
_"JSON-Schema inspired schema syntax"_

How is the syntax different from pure JSON-Schema? I'm unclear on this point.

lzehl commented:
The openMINDS schema syntax is not the same as JSON-Schema.

apdavison commented on instances/latest/contentTypes/application_vnd.openminds.schema+json.jsonld:
```diff
@@ -0,0 +1,19 @@
+  "fileExtension": [
+    ".schema.omi.json"
+  ],
+  "name": "application/vnd.openminds.schema+json",
```
how about "application/vnd.openmetadatainitiative.openminds.schema+json" ?

lzehl replied:
okay with me as well

lzehl commented:
@apdavison @olinux @openMetadataInitiative/openminds-developers I've updated this PR according to the namespace change, updated the instance and bit and added an instance for the openMINDS schema template as well. Before I populate across versions, please provide your feedback/approval for these changes.

apdavison commented:
I think we should use "openmetadatainitiative" rather than "om-i". Other than that, it looks fine.

lzehl commented:
> I think we should use "openmetadatainitiative" rather than "om-i". Other than that, it looks fine.

@apdavison done

lzehl commented:
@apdavison could you double check if the content types are okay now?
